.storybook/decorators/PropCombinations/README.md

```diff
@@ -0,0 +1,275 @@
+# Prop Combinations Decorator
+
+## Motivation
```

Comment:
Can we add documentation for argTypes too?

Reply:
? There's no `argTypes` property specific to the decorator

Reply:
`argTypes` isn't used in the decorator at all. I've only added a convenience type in this PR